incubating/bump-build-number/codefresh.yml

```diff
@@ -0,0 +1,11 @@
+version: '1.0'
+steps:
+  BuildImage:
+    title: Building Docker Image
+    type: build
+    image_name: codefreshplugins/bump-build-number
+  PushingToRegistry:
+    type: push
+    title: Pushing To Registry
+    candidate: ${{BuildImage}}
+    tags: [ "latest", "1.0.0" ]
```

Comment:
@brandonphillips please add a conditional around this push for master branch only.

Reply:
Custom annotation name is there to allow them to track the build number for multiple branches and not just master

Reply:
Done - please verify.